### PR TITLE
Lower levenshtein tolerance to 2

### PR DESCRIPTION
**Pull request by danfinlay**
To reduce false-positives of the phishing detector.

Otherwise [defaults to 3](https://github.com/MetaMask/eth-phishing-detect/blob/master/src/detector.js#L2)

### CODE REVIEW

**Review by brad-decker — APPROVED** [review]

**Comment by Gudahtt** [comment]
Could we go through the whitelist and remove any that are no longer blocked by the fuzzy list as well? This change will likely mean a lot of those are no longer needed in the list.

**Comment by Gudahtt** [comment]
We could probably add a CI step to check for redundant entries :thinking:

File: src/third-party/PhishingController.ts [hunk 1]
```diff
@@ -75,6 +76,7 @@ export class PhishingController extends BaseController<PhishingConfig, PhishingS
     this.defaultState = {
       phishing: DEFAULT_PHISHING_RESPONSE,
       whitelist: [],
+      tolerance: 2,
     };
     this.detector = new PhishingDetector(this.defaultState.phishing);
```
Comment (Gudahtt) [inline src/third-party/PhishingController.ts:81]: The `tolerance` value needs to be passed into the constructor here, via this `phishing` property. Maybe we could add this to `DEFAULT_PHISHING_RESPONSE`? I don't think this is working as currently written.

**Review by Gudahtt — CHANGES_REQUESTED** [review]

File: src/third-party/PhishingController.ts (hunk 1 above)
Comment (kumavis) [reply src/third-party/PhishingController.ts:81]: prolly should just be a default over on eth-phishing-detect?

**Comment by danfinlay** [comment]
> Could we go through the whitelist and remove any that are no longer blocked by the fuzzy list as well? 

Since the fuzzy list operates on a tolerance parameter, I don't think we can generically cleanse the whitelist of exceptions. I guess we could sanitize for the lowest-tolerance parameter, like 1?

**Comment by Gudahtt** [comment]
>Since the fuzzy list operates on a tolerance parameter, I don't think we can generically cleanse the whitelist of exceptions. I guess we could sanitize for the lowest-tolerance parameter, like 1?

Fair enough, though perhaps we can set a max tolerance as well? Then require everything on the whitelist to be matched by _some_ valid tolerance.

**Pull request closed without merging**